azkaban-web-server/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -517,6 +517,13 @@ private void ajaxFetchExecFlowLogs(HttpServletRequest req,
     } catch (ExecutorManagerException e) {
       throw new ServletException(e);
     }
+
+    /*
+     * We originally consider leverage Drop Wizard's Timer API {@link com.codahale.metrics.Timer}
+     * to measure the duration time.
+     * However, Timer will result in too many accompanying metrics (e.g., min, max, 99th quantile)
```

Comment:
Depending on the report interval, the timer may provide more accurate measurement. e.g. if there are many events happen in one minute and the reporting interval is 1 minute, only the last one is reported. 
http://metrics.dropwizard.io/3.1.0/apidocs/com/codahale/metrics/Timer.html
We can try this measure for now and re-evaluate later.